internal/apex/wg-iface.go

```diff
@@ -43,21 +46,25 @@ func setupDarwinIface(logger *zap.SugaredLogger, localAddress, dev string) error
 	_, err := RunCommand("wireguard-go", dev)
 	if err != nil {
 		logger.Errorf("failed to create the %s interface: %v\n", dev, err)
+		return fmt.Errorf("%v", interfaceErr)
```

Comment:
Guess if you wanted to return other errors out of that method it could be returned via the error type and not specified as interfaceErr. Also interfaceErr isn't a type. Whatever works for me.

FYI, I apparently pushed a binary in a patch, added those as deleted in this PR lol :sweat_smile: